lib/ratyrate/helpers.rb

```diff
@@ -41,34 +41,43 @@ def rating_for(rateable_obj, dimension=nil, options={})
                   false
                 end
 
+    rating   = if options.has_key?(:rating)
```

Comment:
Manual star is a good idea. What't difference in stars and rating? also this rating variable isn't used anywhere.

this might be better way to pass stars manually.

```
 starts = if options[:stars]
                options[:stars]
             elsif user  = options[:user]
               rating = user.ratings_given.where(rateable: rateable_obj, dimension: dimension).first
               rating ? rating.stars : 0
             else
               cached_average = rateable_obj.average dimension
               cached_average ? cached_average.avg : 0
             end
```